crates/pyrefly_bundled/generate_docstubs.py

```diff
@@ -0,0 +1,42 @@
+# /// script
+# requires-python = ">=3.11"
+# dependencies = [
+#   "docify",
+# ]
+# ///
+
+from __future__ import annotations
+
+from pathlib import Path
+from shutil import copytree, rmtree
+
+from docify import main as docify  # type: ignore
```

Comment:
which type checker are you ignoring diagnostics on here? is it pyrefly? can we vendor docify so we don't need to pip install on the client?